Source/WebCore/Modules/mediasource/ManagedMediaSource.cpp

```diff
@@ -77,8 +77,8 @@ void ManagedMediaSource::setStreaming(bool streaming)
         return;
     ALWAYS_LOG(LOGIDENTIFIER, streaming);
     m_streaming = streaming;
-    if (m_private)
-        m_private->setStreaming(streaming);
+    if (RefPtr msp = protectedPrivate())
```

Comment:
RefPtr msp = m_private works too.

Reply:
yes, but I figured it looked nicer that way